tools/flight_data_downloader/fdd_transport.py

```diff
@@ -0,0 +1,43 @@
+import os
+from print_colors import print_red
+
+current_output_folder = os.getcwd()
+
+class FDDTransport:
+
+    def __init__(self):
+        pass
+
+    def _get_file(self, file: str) -> bytes:
+        print_red("FDDTransport is an abstract class")
+
+    def tree(self):
+        contents = self._get_file("tree")
+        if contents is not None:
+            print(contents)
+        else:
+            print_red("Error downloading tree file.")
+
+    def download(self, file: str):
+        file_name = file.split("/")[-1]
+        out_path = os.path.join(current_output_folder, file_name)
+
+        with open(out_path, "w") as fout:
```

Comment:
i think you want "wb" here since we want to write binary data to the file since we're returning bytes from _get_file()